theseus_gui/src/components/ui/AccountsCard.vue

```diff
@@ -214,13 +215,23 @@ const handleClickOutside = (event) => {
   }
 }
 
+const unlisten = await process_listener(async (e) => {
+  if (e.event === 'launched') {
+    await refreshValues()
+  }
+})
+
```

Comment:
what is this used for?

Reply:
After launch happens, (and the refreshing of access tokens,etc, the launcher might update its cache of account names if those names happen to have changed, and the accoutns card should reflect those)

See #598 
.